docs/topics/dummy-data.md

```diff
@@ -0,0 +1,100 @@
+# How to setup dummy data
+
+## docker-compose
+
+When starting with a fresh new checkout of the project it is possible to start-up the API using
+`docker-compose up -d api`. This will start the API using the 
+[scripts/initialize.sh](../../scripts/initialize.sh) script.
```

Comment:
As above, I would like to have the option to not fully bootstrap SIA in this case.